plan/plans/show_test.go

```diff
@@ -63,6 +63,23 @@ func (p *testShowSuit) SetUpSuite(c *C) {
 	variable.BindSessionVars(p)
 }
 
+func (p *testShowSuit) TestCommon(c *C) {
```

Comment:
This test tests several types of show plans, and they are not common for all show plans.
I think rename it to `TestSimple` would be better.